lib/Jejik/MT940/Parser/Ing.php

```diff
@@ -224,8 +224,11 @@ protected function code(array $lines): ?string
      */
     protected function eref(array $lines): ?string
     {
-        if (isset($this->codeWords($lines)[self::CODEWORD_EREF])) {
-            return $this->codeWords($lines)[self::CODEWORD_EREF];
+        if (isset($this->codeWords($lines)[self::CODEWORD_EREF]) || isset($this->codeWords($lines)[self::CODEWORD_REMI_USTD])) {
+            return sprintf('%s %s',
+                isset($this->codeWords($lines)[self::CODEWORD_EREF]) ? $this->codeWords($lines)[self::CODEWORD_EREF] : null,
+                isset($this->codeWords($lines)[self::CODEWORD_REMI_USTD]) ? $this->codeWords($lines)[self::CODEWORD_REMI_USTD] : null
+            );
```

Comment:
`if (false === isset($this->codeWords($lines)[self::CODEWORD_EREF]) && false === isset($this->codeWords($lines)[self::CODEWORD_REMI_USTD])) {
    return null;
}

return sprintf('%s %s',
                isset($this->codeWords($lines)[self::CODEWORD_EREF]) ? $this->codeWords($lines)[self::CODEWORD_EREF] : null,
                isset($this->codeWords($lines)[self::CODEWORD_REMI_USTD]) ? $this->codeWords($lines)[self::CODEWORD_REMI_USTD] : null
            );`